src/app/src/components/annotations/annotator.tsx

```diff
@@ -607,7 +632,7 @@ export default class Annotator extends Component<
       }
 
       this.setState({ predictDone: 0, uiState: null });
-    }, 750);
+    }, 0);
```

Comment:
What's the reason for this change? If it is going to be 0, I don't think we will need the setTimeout anymore

Reply:
This change was done so that it could "simulate" the call of an API. Since this is not needed anymore, I have removed it.

Reply:
Ok got it!